### PR TITLE
support for (central) scenes

### PR DESCRIPTION
**Pull request by marcelveldt**
Support for both legacy/network scenes and central scenes.

Will fire event zwave_mqtt.scene_activated event in hass

### CODE REVIEW

**Comment by marcelveldt** [comment]
fixes #32

File: custom_components/zwave_mqtt/__init__.py [hunk 1]
```diff
@@ -130,6 +131,12 @@ def async_value_changed(value):
             value.label,
             value.value,
         )
+        # Handle a scene activation message
+        if value.command_class in [
+            "COMMAND_CLASS_SCENE_ACTIVATION",
+            "COMMAND_CLASS_CENTRAL_SCENE",
+        ]:
+            handle_scene_activated(hass, value)
```
Comment (balloob) [inline custom_components/zwave_mqtt/__init__.py:141]: Add a return. No further processing needed. 
```suggestion
            return

```

File: custom_components/zwave_mqtt/__init__.py [hunk 2]
```diff
@@ -161,3 +168,34 @@ def async_value_changed(value):
...
+                scene_id = item["Value"]
+                break
+    _LOGGER.debug(
+        "Scene activated - node: %s - scene_id: %s - label: %s",
+        scene_value.node.id,
+        scene_id,
+        label,
+    )
+    # Simply forward it to the hass event bus
+    hass.bus.fire(
```
Comment (balloob) [inline custom_components/zwave_mqtt/__init__.py]:
```suggestion
    hass.bus.async_fire(
```

**Review by balloob — APPROVED** [review]
ok to merge once test added. 

For tests, copy [this helper](https://github.com/home-assistant/home-assistant/blob/dev/tests/common.py#L1055-L1065) to `tests/common.py`

**Comment by balloob** [comment]
Let's not forget to update the README.

**Comment by marcelveldt** [comment]
> Let's not forget to update the README.

readme updated

**Pull request merged**